core/src/test/scala/org/apache/spark/sql/delta/cdc/CDCReaderSuite.scala

```diff
@@ -145,10 +145,12 @@ class CDCReaderSuite
       writeCdcData(
         log,
         spark.range(30, 35).toDF().withColumn(CDC_TYPE_COLUMN_NAME, lit("update_post")))
-
+      val changes = CDCReader.changesToBatchDF(log, 0, 2, spark)
+      // Just to kickoff `sizeInBytes` function to verify match pattern does not throw an exception
+      changes.queryExecution.analyzed.stats
```

Comment:
can you put this into its own test, and verify the result?

Reply:
Done. Also added a comment saying that the purpose of this test is not the actual calculation of the `queryExecution.analyed/optimized.stats` but rather executing the `sizeInBytes` function inside `TahoeCDCBaseFileIndex`